devd.go

```diff
@@ -237,6 +237,11 @@ func main() {
 		Default("false").
 		Bool()
 
+	excludes := kingpin.Flag("exclude", "Exclude this file from triggering a reload").
```

Comment:
The doc string here can be clearer. "Glob pattern for files to exclude from livereload" or similar.